sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -48,9 +46,8 @@
     private final ConcurrentMap<String, SessionSubscription> sessionMap = new ConcurrentHashMap<>();
 
     private final AtomicBoolean isDisposed = new AtomicBoolean();
-    private final DirectProcessor<AmqpShutdownSignal> shutdownSignals = DirectProcessor.create();
-    private final FluxSink<AmqpShutdownSignal> shutdownSignalsSink = shutdownSignals.sink();
-    private final ReplayProcessor<AmqpEndpointState> endpointStates;
+    private final Sinks.One<AmqpShutdownSignal> shutdownSignalSink = Sinks.one();
+    private final Flux<AmqpEndpointState> endpointStates;
```

Comment:
Why was this changed to a `Flux` and not a `Sinks.many().replay().latest()`?

Reply:
Because this isn't programmatically emitted. On line 104, all of the results come from another flux. `handler.getEndpointStates()`.
https://github.com/Azure/azure-sdk-for-java/pull/19928/files/cfda5e9f016e9d9504f1826173e11b1e88a76d9e#diff-4fd2600dc96ee9531c22be07b35f51eab6a957073c9adff75d0c54440b617ec3R104